src/GlobalData.ts

```diff
@@ -9,14 +9,32 @@
  * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
  */
 
-"use strict";
```

Comment:
Should we add the [`alwaysStrict` compiler flag](https://www.typescriptlang.org/tsconfig#alwaysStrict) to `ts.config`?  It has the same effect as `"use strict"`, more or less.  I also looked at the stackoverflow article  ["Use Strict" needed in a typescript file](https://stackoverflow.com/questions/31391760/use-strict-needed-in-a-typescript-file) and there are some arguments for making sure the transpiled code has the "use strict"; directive in it, which is part of what `alwaysStrict` does.

Reply:
Good point. I'm thinking to set [the strict flag](https://www.typescriptlang.org/tsconfig#strict) to `true`. This documentation says it enables a wide range of type checking behavior including "alwaysStrict". The linting and running of our code base so far seems working well with `strict: true`.

What do you think?

Reply:
Setting `strict: true` introduces the type verification in the imported modules. Some of them don't have a proper type definitions. I ended up with what you suggested of `alwaysStrict: true` so that "use strict" is no longer needed in every source file.